xDripTests/Controllers/Notifications/GlucoseNotificationWorkerTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  GlucoseNotificationWorkerTests.swift
+//  xDripTests
+//
+//  Created by Ivan Skoryk on 02.07.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import XCTest
+@testable import xDrip
+
+final class GlucoseNotificationsWorkerTests: XCTestCase {
```

Comment:
Inherit all unit tests involving Realm from `AbstractRealmTests`. Otherwise previous tests might affect the results.